src/winit_input_helper.rs

```diff
@@ -255,12 +255,13 @@ impl WinitInputHelper {
         }
     }
 
-    /// Returns `0.0` if the mouse is outside of the window.
+    /// Returns `(0.0, 0.0)` if the mouse is outside of the window.
     /// Otherwise returns the amount scrolled by the mouse during the last step.
-    pub fn scroll_diff(&self) -> f32 {
+    /// Returns (vertically, horizontally)
```

Comment:
This should be `(horizontally, vertically)` right?

(Sorry for the long delay in response again)

Reply:
I've swapped them around